### PR TITLE
[Refactor] Settings Page

### PR DESCRIPTION
**Pull request by YashKamboj**
Signed-off-by: Yash Kamboj <yashkamboj29@gmail.com>

**Description**
Screenshot (194)

Screenshot (176)
Screenshot (177)
Screenshot (178)
Screenshot (174)
Screenshot (175)

  

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/Button.js [hunk 1]
```diff
@@ -1,8 +1,30 @@
...
 import { styled } from "@mui/material/styles";
 import { deepSpaceSparkle } from '../styles/colors';
 
 export const MetricsButton = styled(Button)(({theme})=>({
     backgroundColor: deepSpaceSparkle,
-    width: "240px",
 }))
+
+
+ const RightAlignButtonContainer = styled(Box)(({theme})=>({
```
Comment (sudo-NithishKarthik) [inline ui/components/Button.js:10]: `Container` is being used differently in our codebase, let's use a different name for this purpose. Maybe `wrapper`?

File: ui/components/SettingsComponents/GrafanaComponent.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React, {useState} from 'react'
```
Comment (sudo-NithishKarthik) [inline ui/components/SettingsComponents/GrafanaComponent.js:1]: Its a good decision that you've moved this into `SettingsComponents` directory 👍

**Comment by leecalcote** [comment]
@YashKamboj will you change the label "Service Meshes" to "Adapters"?

**Comment by YashKamboj** [comment]
@leecalcote I am not able to find an appropriate icon for that tab , your help would be needed.

**Comment by YashKamboj** [comment]
@sudo-NithishKarthik a review??

**Comment by leecalcote** [comment]
@YashKamboj service mesh icon - https://layer5.io/static/cb310234b6631abcabb632a85974a3dd/service-mesh.svg

**Comment by YashKamboj** [comment]
@leecalcote the icon look too small
Screenshot (200)

**Comment by leecalcote** [comment]
Please note that "Service Meshes" as the tab label is now "Adapters".

**Comment by YashKamboj** [comment]
@leecalcote i changed that, just haven't updated the screenshot

**Review by Chadha93 — COMMENTED** [review]
@YashKamboj done with the all changes on this pr? Could you please provide a ss/video of the latest update here?

**Comment by YashKamboj** [comment]
@Chadha93 i have made all the changes, just the icon of the adapters tab, we can add that later

File: ui/components/SettingsComponents/MeshAdapterConfigComponent.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import ReactSelectWrapper from "@/components/ReactSelectWrapper"
+import {RightAlignButton} from "@/components/Button"
+
+function MeshAdapterConfigComponent() {
+  const theme = useTheme();
+
+  return (
+    <Box sx={{p: theme.spacing(4)}}>
+       <AdaptersListContainer>{(props) => <AdaptersChipList {...props} />}</AdaptersListContainer>
+         <ReactSelectWrapper style={{ marginTop: theme.spacing(4)}} label="Mesh Adapter UR" />
```
Comment (Chadha93) [inline ui/components/SettingsComponents/MeshAdapterConfigComponent.js:14]: sx for style??

File: ui/components/SettingsComponents/MeshConfigComponent.js [hunk 4]
```diff
@@ -0,0 +1,495 @@
...
+                          <List>
+                            <ListItem>
+                              <Tooltip
+                                title={operatorState
+                                  ? `Version: ${operatorVersion}`
+                                  : "Not Available"}
+                                aria-label="meshSync"
+                              >
+                                <Chip
+                                  // label={inClusterConfig?'Using In Cluster Config': contextName + (configuredServer?' - ' + configuredServer:'')}
```
Comment (Chadha93) [inline ui/components/SettingsComponents/MeshConfigComponent.js:344]: clear this if not required.

File: ui/components/SettingsComponents/MeshAdapterConfigComponent.js (hunk 3 above)
Comment (YashKamboj) [reply ui/components/SettingsComponents/MeshAdapterConfigComponent.js:14]: this is not a mui component, we can only use sx in mui components

**Review by Chadha93 — APPROVED** [review]
Good work @YashKamboj :1st_place_medal:

**Pull request merged**